### PR TITLE
Create plugin opsgenie

### PR DESCRIPTION
**Pull request by bkaganyildiz**
@slide maintainer
 @daniel-beck maintainer 
would you accept the pr. thanks in advance 

# Description
[OpsGenie Jenkins Plugin Github](https://github.com/jenkinsci/opsgenie-plugin)
[Hosting Request](https://issues.jenkins-ci.org/browse/HOSTING-381)
This plugin allows posting build related alerts to [OpsGenie](https://www.opsgenie.com/) that notify who is on call.

# Submitter checklist for changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

- [x] Add link to resolved HOSTING issue in description above

### For a new permissions file only

- [x] Make sure the file is created in `permissions/` directory
- [x] `artifactId` (pom.xml) is used for `name` (permissions YAML file).
- [x] [`groupId` / `artifactId` (pom.xml) are correctly represented in `path` (permissions YAML file)](https://github.com/jenkins-infra/repository-permissions-updater/#managing-permissions)
- [x] Check that the file is named `plugin-${artifactId}.yml` for plugins

### When adding new uploaders (this includes newly created permissions files)

- [x] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [x] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### CODE REVIEW

File: permissions/plugin-opsgenie.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: "opsgenie"
+paths:
+- "com/opsgenie/integration/opsgenie"
+- "org/jenkins-ci/plugins/opsgenie"
```
Comment (daniel-beck) [inline permissions/plugin-opsgenie.yml]: This seems to be unused. Are there plans to use this path or why is it included?

Comment (bkaganyildiz) [inline permissions/plugin-opsgenie.yml]: There were two groupId in pom.xml so I could not be sure which one is the right one.

Comment (daniel-beck) [inline permissions/plugin-opsgenie.yml]: Only the one you use for your plugin matters, the parent POM's does not.

Comment (bkaganyildiz) [inline permissions/plugin-opsgenie.yml]: Ok. I'm updating it.

**Pull request merged**